packages/parse-cli-args/test/index.ts

```diff
@@ -207,13 +207,59 @@ test('any unknown command is treated as a script', async () => {
     fallbackCommand: 'run',
     getCommandLongName: () => null,
     universalOptionsTypes: { filter: [String, Array] },
-  }, ['foo', '--recursive'])
+  }, ['--recursive', 'foo'])
```

Comment:
Had to rearrange the `--recursive` flag here to fix the test after these changes. This was expected from the behavior change described in the pull request description.